HeavyIonsAnalysis/JetAnalysis/python/FullJetSequence_nominalPP.py

```diff
@@ -13,23 +13,21 @@
 from RecoJets.JetProducers.PFJetParameters_cfi import *
 from RecoJets.JetProducers.AnomalousCellParameters_cfi import *
 akSoftDrop4PFJets = cms.EDProducer(
-    "FastjetJetProducer",
+    "SoftDropJetProducer",
```

Comment:
No, it is a new producer which I wrote. It gives additional functionality with respect to the SoftDrop one can run in FastJetProducer:
- stores zg in the event
- stores the number of dropped branches in the event
- can run SoftDrop on only the charged constituents of the jet